client/signup/step-wrapper/index.jsx

```diff
@@ -208,7 +237,9 @@ class StepWrapper extends Component {
 			'is-large-skip-layout': isLargeSkipLayout,
 			'has-navigation': hasNavigation,
 		} );
-		const enablePresales = flows.getFlow( flowName, this.props.userLoggedIn )?.enablePresales;
+
+		const flow = flows.getFlow( flowName, this.props.userLoggedIn );
+		const enableHelpCenter = flow?.enableHelpCenter && isEnabled( 'signup/help-center-link' );
```

Comment:
I would move this flag check to the prop. So that deploying is just flipping a flag.